lib/widgets/topic_list.dart

```diff
@@ -169,8 +169,12 @@ class _TopicListState extends State<_TopicList> with PerAccountStoreAwareStateMi
       return const Center(child: CircularProgressIndicator());
     }
 
-    // TODO(design) handle the rare case when `lastFetchedTopics` is empty
-
+    if (lastFetchedTopics!.isEmpty) {
+      final zulipLocalizations = ZulipLocalizations.of(context);
+      return PageBodyEmptyContentPlaceholder(
+        header: zulipLocalizations.topicListEmptyPlaceholderHeader);
+    }
+    
```

Comment:
nit: stray whitespace at end of line
```suggestion

```

Git should highlight this for you when you self-review your changes, with `git log -p`. Here's how it looks for me:
<img width="534" height="376" alt="Image" src="https://github.com/user-attachments/assets/2ba9633b-68c1-4846-9c54-83e584f29942" />

I believe it does that by default. In any case, if it's not doing it for you, the `#git help` channel on chat.zulip.org would be a great place to ask.